docs/visualization_guide.md

```diff
@@ -0,0 +1,92 @@
+# Visualizing and Debugging Your Knowledge Graph
+
+The following step-by-step guide walks through the process to visualize a knowledge graph after it's been constructed by graphrag. Note that some of the settings recommended below are based on our own experience of what works well. Feel free to change and explore other settings for a better visualization experience!
+
+## 1. Run the Pipeline
+Before building an index, please review your `settings.yaml` configuration file and ensure that graphml snapshots is enabled
+```yaml
+snapshots:
+  graphml: true
+```
```

Comment:
I would add an optional block to generate x/y positions for the nodes, which can be used in other tools:

```
embed_graph:
  enabled: true
umap:
  enabled: true
```

Reply:
Done. Let me know if you think the wording could be better.